src/generators/static/tsConfigFileGenerator.ts

```diff
@@ -15,7 +25,7 @@ export function generateTsConfig(project: Project) {
       esModuleInterop: true,
       allowSyntheticDefaultImports: true,
       forceConsistentCasingInFileNames: true,
-      lib: ["es6", "dom"],
+      lib: ["es6", "dom", ...esNext],
```

Comment:
This could cause problems to the users because this way `esnext` is in the compiled sources which brings all their definitions into scope at runtime even if the user does not want to use it. I think a better approach is to include the `asyncIterable` package only in compiled sources by having a triple slash directive in `index.ts` file (e.g. [here](https://github.com/Azure/azure-sdk-for-js/blob/2c2e35f6f986afbf7f1a6568678e0a89a4b59a4c/sdk/textanalytics/ai-text-analytics/src/index.ts#L4)).

Reply:
Thanks for bringing this up! I'm fixing it